File: algorithm/ceil.py
# https://atcoder.jp/contests/abc207/tasks/abc207_b

A, B, C, D = map(int, input().split())

def ceil(a, b):
    return (a + b - 1) // b

d = C * D - B
if d <= 0:
    ans = -1
else:
    ans = ceil(A, d)
print(ans)

File: algorithm/ceil.cc
// https://atcoder.jp/contests/abc239/tasks/abc239_b

#include <iostream>
using namespace std;

long long ceil(long long a, long long b) {
  return (a + b - 1) / b;
}

int main() {
  long long X;
  cin >> X;
  long long ans;
  if (X > 0) {
    ans = X / 10;
  } else if (X < 0) {
    ans = -ceil(-X, 10);
  } else {
    ans = 0;
  }
  cout << ans << endl;
  return 0;
}
